e2e/pages/app/app.module.ts

```diff
@@ -1,10 +1,11 @@
 import { NgModule } from '@angular/core';
```

Comment:
In styles.css there are the following styles:

```css
body {
  padding: 15px;
}

h1 {
  color: #369;
  font-family: Arial, Helvetica, sans-serif;
  font-size: 250%;
}
```

These should be removed as they are affecting the appearance of things within the application unnessecarily